admin/cron.php

```diff
@@ -67,15 +67,15 @@ function logs($s) {
 if(date('G') == 5) { // 5 hodin ráno
   logs('Zálohuji databázi.');
 
-  if(!defined('ZALOHA_DB_SLOZKA'))
-    throw new Exception('Není definována konstanta s adresářem pro zálohování.');
-
-  $db = dbConnect();
-  $dump = new MySQLDump($db);
-  $time = date('Y-m-d_His');
-  $dump->save(ZALOHA_DB_SLOZKA . "/export_$time.sql.gz");
-
-  logs('Záloha dokončena.');
+  if(!defined('ZALOHA_DB_SLOZKA') || !ZALOHA_DB_SLOZKA) {
```

Comment:
Tak ono by ještě krásnější bylo, kdybychom kontrolovali i existenci toho adresáře, ale asi to s kontrolou nemusíme přehánět.

Reply:
Máš recht. Hlavně by bylo fajn hlásit, že nám zálohy DB selhaly, abychom to nezjistili až bude pozdě.

Hlásím to nově mailem.